src/components/Stepper/StepperSidebar/StepperSidebar.tsx

```diff
@@ -5,26 +5,27 @@ import { Fragment, useState } from "react";
 import Tick from "@/components/AppIcons/Tick";
 import { cn } from "@/utils/helper";
 import Return from "@/components/AppIcons/Return";
-import { IStepperSidebar } from "@/components/Stepper/StepperSidebar/StepperSidebar.d";
+import { useStepperContext } from "@/context/StepperContext";
 
-const StepperSidebar = (props: IStepperSidebar) => {
-  const { steps, className, activeStepIndex } = props;
+const StepperSidebar = () => {
+  const { stepperStepInformation: steps, activeStepIndex } =
+    useStepperContext();
   const [hoveredIndex, setHoveredIndex] = useState(-1);
   const completedPercentage = ((activeStepIndex + 1) / steps.length) * 100;
 
   return (
-    <div className={cn("h-full bg-primary", className)}>
+    <div className="order-1 h-full lg:order-2">
       {/* LG version */}
       <ul
-        className="sticky top-[100px] hidden p-16 text-white lg:block"
+        className="sticky top-[100px] hidden px-14 py-16 text-white lg:block"
         aria-label="progress"
       >
         {steps.map((step, index) => {
           return (
-            <Fragment key={step.title}>
+            <Fragment key={step.id}>
```

Comment:
Let's not rely on only `id`. Add some wording so that it can be completely unique